.changeset/cold-dots-buy.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin-theme": minor
```

Comment:
The theme package has been merged into the admin package. Please update the changeset.